lib/registries/addon/branded/discover/route.ts

```diff
@@ -14,7 +15,11 @@ export default class BrandedRegistriesDiscoverRoute extends Route {
     afterModel(provider: RegistrationProviderModel) {
         if (!provider.brandedDiscoveryPage) {
             if (provider.id === 'osf') {
-                this.transitionTo('discover');
+                this.router.transitionTo('search', {
+                    queryParams: {
+                        resourceType: 'osf:Registration',
+                    },
+                });
             } else {
                 this.transitionTo('page-not-found', notFoundURL(window.location.pathname));
             }
```

Comment:
In the `buildRouteInfoMetadata` below, we should also add `providerId: this.model.id,`